R/graph_embedding.R

```diff
@@ -70,4 +70,12 @@ embedGraphUmap <- function(graph, verbose=T, min.prob=1e-3, min.visited.verts=10
     return(list(adj.info=adj.info, commute.times=commute.times, umap=umap))
 
   return(umap)
-}
\ No newline at end of file
+}
+
+findSubcommunities <- function(con, cluster, annot=NULL, ...) {
```

Comment:
This function should be in conos.R, as it has nothing to do with graph embeddings.
Also, it should be consistent with the other functions. Which means that if should accept parameters `group, clustering=NULL, groups=NULL`. See [`getDifferentialGenes`](https://github.com/rrydbirk/conos/blob/dev/R/conclass.R#L332) as an example.

Reply:
Indeed, I'd expect that `findSubcommunities` function extend the existing annotation instead of returning only the part, which corresponds to the subdivided cluster. Which mean adding new levels to the old annotation.

Reply:
I've tried to correct this. I don't know if this is transferable to "walktrap", do you know? I have almost entirely worked with Leiden.

The function now appends new annotation to original annotation. Do you have any good ideas for naming new clusters? Right now, it is possible to create new clusters that will have the same name as original clusters.